### ISSUE TITLE
No response issue at new finger touch when full screen

### ISSUE THREAD

**Issue by GoogleCodeExporter** [JustLoy77/ebookdroid#447]
```
What steps will reproduce the problem?
1.set Full screen
2.back to pdf
3.touch the screen and move your finger, stop, touch and move again

What is the expected output?
always response normally at any touch (moving page)

What do you see instead?
Response only after your new touch, or new touch not move page at all.

What version of the product are you using?
Ebookdroid 1.64 or 1.66 featured one.

On what operating system?
JB 4.12

What device are you using?
MOTOROLA ME865

Please provide any additional information below.

I guess I am not the first one to report the problem, and I was expecting the 
new version to shoot it, but the new version of 1.66 has no change on this 
issue.
```

Original issue reported on code.google.com by `chqpushm...@gmail.com` on 6 Feb 2013 at 6:41

**Comment by GoogleCodeExporter** [comment]
```
All questions to Google. It is default behavior of Android OS
```

Original comment by `Alexander.V.Kasatkin@gmail.com` on 6 Feb 2013 at 7:45
- Changed state: **WontFix**

**Comment by GoogleCodeExporter** [comment]
```
Other app doesn't have this kind of issue?

```

Original comment by `viktre.w...@gmail.com` on 18 May 2014 at 10:28
